backend/src/services/import_service.py

```
@@ -0,0 +1,326 @@
+import logging
+import json
+import zipfile
+import io
+from typing import Dict, Any, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)